pkg/render/manager.go

```diff
@@ -21,9 +21,8 @@ import (
 
 	tigerakvc "github.com/tigera/operator/pkg/render/common/authentication/tigera/key_validator_config"
 
-	"github.com/tigera/operator/pkg/render/common/authentication"
-
 	ocsv1 "github.com/openshift/api/security/v1"
+	"github.com/tigera/operator/pkg/render/common/authentication"
```

Comment:
I was not sure where to place this comment, but I don't really see any change that makes components reach kibana through voltron. Does this change exist solely in the voltron pod? Shouldn't there be some env set in here to dictate this?